src/components/Editor/Footer.js

```diff
@@ -199,6 +198,14 @@ class SourceFooter extends PureComponent<Props> {
   }
 }
 
+const {
```

Comment:
this could be done on line 9, since actions are not being used anywhere but here